proxy.py

```diff
@@ -2,7 +2,7 @@
 import socket
 import json
 import datetime, time
-import ConfigParser
+import http
```

Comment:
I had defined some method in http.py.
And need call the method with http.method(***)